getting-started.md

```diff
@@ -0,0 +1,102 @@
+---
+title: "Getting started"
+description: Get started with Coder.
+---
+
+We’re excited for you to be part of the growing community of Coder users, and we
+wanted to provide an onboarding guide to ensure you have a great experience.
+Coder’s [documentation is completely open-source](https://github.com/cdr/docs),
+so please feel free to contribute and provide feedback as you desire.
+
+## Setup and deploy Coder
+
+To get started with Coder, you’ll need to [deploy Coder](./setup/index.md) to a
+[Kubernetes cluster](./setup/kubernetes/index.md). We have documentation to
+assist you with [creating a cluster](./setup/kubernetes/index.md) if needed.
+Once you have a cluster, you can [install Coder via
+Helm](./setup/installation.md).
+
+## Configure Coder
+
+After you deploy Coder, you’ll need to [upload your license
+file](./setup/configuration.md) before you can configure the application. Once
```

Comment:
Perhaps mention "You can get a license for free [here](https://coder.com/trial)"

The product already mentions this and it's very likely trial users will be coming to this doc, but I don't want to make it seem like a paid license is required to try Coder.